docusaurus/i18n/de/docusaurus-plugin-content-docs/current/getting-started/configuration/game-settings.md

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_position: 2
+---
+
+# Spiel-Einstellungen
+
+Ändern von Spiel-Einstellungen mit Umgebungsvariablen.
+
+## Mit Umgebungsvariablen
+
+:::warning
+Diese Umgebungsvariablen und Einstellungen können sich ändern, da das Spiel noch in der Beta-Phase ist.
+
+Überprüfen Sie die [offizielle Webseite für unterstützte Parameter.](https://tech.palworldgame.com/optimize-game-balance)
+:::
+
+Das Konvertieren von Servereinstellungen in Umgebungsvariablen folgt denselben Prinzipien (mit einigen Ausnahmen):
```

Comment:
Das liest sich komisch. Vllt sowas wie:

"Um Servereinstellung in Umgebungsvariablen zu konvertieren müssen folgende Regeln eingehalten werden:"